### ISSUE TITLE
Open additional windows from concordance navigator

### ISSUE THREAD

**Issue by roewenstrunk** [Edirom/edirom-concordance-navigator#12]
Give the user the opportunity to open additional sources or texts from the concordance navigator that are currently not visible

**Comment by DennisFriedl** [comment]
Yes, I have a "Open connected documents" button already in mind. This logic already works in Edirom Mobile and just needs to be implemented in Edirom Desktop.
